index.html

```diff
@@ -94,9 +96,11 @@ <h1 class="title_hiring"> We're hiring </h1>
                     <img class="logo" src="./images/picture-logo.png" width="60px" height="60px">
                 </div>
             </div>
-            <li class="company-description">Foundation is a website template
-                for<br>startups and small teams. It helps to<br>build a website in
-                no time.</li>
+            <div class="footer-column-for-description">
```

Comment:
Why do you need `div` and `span` around the text?

Reply:
I tried to do without the "div" but I couldn't indent like in the design

Reply:
there was 'li' I replaced it with 'span'